README.md

```diff
@@ -47,6 +47,9 @@ guarddog pypi scan requests --exclude-rules exec-base64
 # Scan a local package
 guarddog pypi scan /tmp/triage.tar.gz
 
+# Scan a local directory
```

Comment:
Can we be more specific about what we expect the directory to contain? (1 .tar.gz per package?)

Reply:
Done